### PR TITLE
Add toArray() sanitizer (#679)

### PR DESCRIPTION
**Pull request by soulchild**
This pull request adds a toArray() sanitizer as discussed in #679.

### CODE REVIEW

File: src/chain/sanitizers-impl.spec.ts [hunk 1]
```diff
@@ -103,3 +104,28 @@ describe('#customSanitizer()', () => {
...
+    sanitizers.toArray();
+
+    const meta: Meta = { req: {}, location: 'body', path: 'foo' };
+    const toArray = context.addSanitization.mock.calls[0][0] as CustomSanitizer;
+
+    expect(toArray([], meta)).toEqual([]);
+    expect(toArray('foo', meta)).toEqual(['foo']);
+    expect(toArray(['foo'], meta)).toEqual(['foo']);
+    expect(toArray('', meta)).toEqual(['']);
+    expect(toArray(null, meta)).toEqual([null]);
```
Comment (gustavohenke) [inline src/chain/sanitizers-impl.spec.ts]: Do you see falsy values/nulls becoming an issue when they aren't transformed into `[]`?

Comment (gustavohenke) [reply src/chain/sanitizers-impl.spec.ts]: For clarification: I wonder if other users won't ask for a fine-grained control like in `optional`.

But nonetheless, I'm fine with whatever 👍

Comment (soulchild) [reply src/chain/sanitizers-impl.spec.ts]: I was wondering the same thing while implementing the test cases and didn't come to a conclusion on that. You're talking about transforming `[4,5,null,'',6]` to `[4,5,6]` by means of specifying some sort of `checkFalsy` and/or `checkNull` modifiers, right? I think we could always add options for that at a later time if the need arises. What do you think?

Comment (gustavohenke) [reply src/chain/sanitizers-impl.spec.ts]: > You're talking about transforming `[4,5,null,'',6]` to `[4,5,6]` by means of specifying some sort of `checkFalsy` and/or `checkNull` modifiers, right?

Not that, but whether `null` should become `[]` (like `undefined`), and then that could evolve even crazier... but oh well.

As I said, this code looks fine to me, so will merge it.

**Comment by gustavohenke** [comment]
Hello @soulchild, sorry for the delay. This is now in [`v6.0.0`](https://github.com/express-validator/express-validator/releases/tag/v6.0.0) 🚢

**Pull request merged**